astropy/time/core.py

```diff
@@ -139,6 +139,18 @@ def __init__(self, bound=False):
                                      'yaml': 'jd1_jd2',
                                      None: 'jd1_jd2'}
 
+    def get_sortable_arrays(self):
+        """
+        Return a list of arrays which can be lexically sorted to represent
+        the order of the parent column.
+
+        :return: list of ndarray
```

Comment:
this is super nit picky, and don't need to be changed right now, but longer term it would be nice to use numpydoc format. If you still use pycharm, the docstring format can be changed in preferences-->tools-->python integrated tools

Reply:
Agreed, done.  Thanks for the hint in setting this preference. I had this set before but I think I accidentally blew away my `.idea` directory at some point with `git clean -fxd`, so the docstring format got lost.

Reply:
got bitten by that, too, now I only have `git clean -dfx -e .idea -e notes` in my history and stick to use it from there rather than typing it in :)